tests/driver_ltc4150/main.c

```diff
@@ -45,6 +45,7 @@ static const ltc4150_recorder_t test_recorder = {
 static kernel_pid_t target_pid;
 static char busy_thread_stack[THREAD_STACKSIZE_DEFAULT];
 static ltc4150_dev_t ltc4150;
+static int change_of_load_level;
```

Comment:
I know BSS is initialized with 0, but this rings all the "possible uninitialized value" bells in my head with the access in the main. Could you please add a `= 0` to calm those bells ;-)?